pkg/rosa/versions.go

```diff
@@ -41,10 +41,27 @@ func ScheduleControlPlaneUpgrade(client *ocm.Client, cluster *cmv1.Cluster, vers
 		ScheduleType(cmv1.ScheduleTypeManual).
 		Version(version).
 		NextRun(nextRun).
+		EnableMinorVersionUpgrades(true).
 		Build()
 	if err != nil {
 		return nil, err
 	}
+
+	versionGates, err := client.GetMissingGateAgreementsHypershift(cluster.ID(), upgradePolicy)
+	if err != nil {
+		return nil, err
+	}
+
+	if !ack && len(versionGates) > 0 {
+		return nil, fmt.Errorf("version gate acknowledge is required %v", versionGates[0])
```

Comment:
you should expose all the version gates that needs acknowledge here, not only the first one. Otherwise the user may be thinking that there is only one gate he will acknoledge

Reply:
done. format the output and add all version gates